packages/expo-firebase-core/README.md

```diff
@@ -0,0 +1,38 @@
+
+# expo-firebase-core
+
+Core support for the native Google Firebase SDK. This library ensures that Google Firebase is initialized natively and makes it possible to use 
```

Comment:
```suggestion
Core support for the native Google Firebase SDK. This library ensures that Google Firebase is initialized natively and makes it possible to use native Firebase features from other unimodules.
```

(fixed unnecessary newline)

Reply:
Updated the description altogether to be more inline with the sdk-docs:

> This module provides access to the Firebase configuration and performs initialisation
of the native Firebase App. Unimodules that want to use Firebase can use
this module to get safe access to the native Firebase App.